src/service/action-impl.js

```diff
@@ -446,6 +446,35 @@ export class ActionService {
     return !!this.findAction_(target, actionEventType, opt_stopAt);
   }
 
+  /**
+   * Checks if the given element's registered action resolves to at least one
+   * existing element by id or a global target (e.g. "AMP").
+   * @param {!Element} target
+   * @param {string} actionEventType
+   * @param {!Element=} opt_stopAt
+   * @return {boolean}
+   */
+  hasResolvableAction(target, actionEventType, opt_stopAt) {
```

Comment:
Nit: Rename `target` to `element` to disambiguate with ActionInfo.target on line 462.